projects/ui/src/Styles/Features/_redoc.scss

```diff
@@ -79,13 +82,58 @@
         }
         > div.scrollbar-container,
         div[data-role="search:results"] {
-          background: white;
+          min-height: 45px;
+          padding: 0px;
+          margin: 0px;
+          background-color: constants.$marchGrey;
+          li {
```

Comment:
Can we either tag a class name or, simpler, just throw a comment in here to describe what we've hooked on to.

Reply:
Adding a comment - these are the li elements (one for each search result row) in the search result list.